src/templates/static/home.jsx

```diff
@@ -100,7 +100,7 @@ const AccountsTabContent = ({
                         <img className='responsive' src={it.url_for(image)} />
                     </div>
                     <div className={`center-text gr-12 gr-hide gr-show-m ${mobile_class || ''}`}>
-                        <div className='gr-8 gr-centered'>
+                        <div className='gr-10 gr-centered margin-top-10'>
```

Comment:
This'll affect desktop viewports as well, if it is specific to mobile, we can use `gr-10-m` and let `gr-8` be intact.

Reply:
the parent div just shown in the mobile and it is hidden in desktop mood, so I think no need for gr-10-m class